server/sonar-server/src/main/java/org/sonar/server/issue/IssueQueryService.java

```diff
@@ -206,7 +209,7 @@ public IssueQuery createFromRequest(SearchWsRequest request) {
         request.getFileUuids(),
         request.getAuthors());
 
-      builder.createdAfter(buildCreatedAfterFromRequest(session, request, allComponentUuids, effectiveOnComponentOnly));
+      builder.createdAfter(buildCreatedAfterFromRequest(session, request, allComponentUuids));
```

Comment:
yes! the parameter is no more needed because I don't [check](https://github.com/SonarSource/sonarqube/pull/996/files/87428868636259638f95ec3bea1943919b1ad70b#diff-f87e5d5f47e5b29953f41a3683c3aadfL233) this parameter when computing the total number of components. I rely only on parameters: `componentKeys`, `componentUuids`, `components`, `componentRoots`, `componentRootUuids`. 
What I want to check is that you provide exactly one root component in one of this parameters.